map/src/frame/components/poi/PoiTypesDialog.jsx

```diff
@@ -37,6 +37,7 @@ export default function PoiTypesDialog({ dialogOpen, setDialogOpen, width }) {
 
     const classes = useStyles();
     const styles = drawerStyles();
+    const activePoiFilters = removeUnusedFilters(ctx.poiCategory?.filters);
```

Comment:
Will crash (inside function and later on activePoiFilters.map) if no ctx.poiCategory.filters exist (undefined).